### ISSUE TITLE
Volume slider on Media Player Tile

### ISSUE THREAD

**Issue by oneseventhree** [resoai/TileBoard#401]
Hi,

is there a way to get a volume slider or just volume in general on the media player tile?

**Comment by akloeckner** [comment]
I have it. Sometimes. Sometimes only volume up and down. For the same device type (KODI). I believe, it's a matter of supported features reported by HA. And somewhere in the chain, it's buggy. For me it seems to be caused in HA already, as the same devices show differently in Lovelace as well...

Short answer, if your device supports setting a volume, the slider should show up automatically.

**Comment by oneseventhree** [comment]
My slider does show up on HA:
<img width="441" alt="Screen Shot 2020-09-03 at 8 23 46 am" src="https://user-images.githubusercontent.com/15043282/92043526-88ed2200-edbf-11ea-9a73-259d59135be9.png">

<img width="379" alt="Screen Shot 2020-09-03 at 8 23 59 am" src="https://user-images.githubusercontent.com/15043282/92043553-930f2080-edbf-11ea-9b5f-aa05aaca3c06.png">


On my tile it shows up like:
<img width="355" alt="Screen Shot 2020-09-03 at 8 23 35 am" src="https://user-images.githubusercontent.com/15043282/92043580-a0c4a600-edbf-11ea-9719-2ce19b124ec6.png">

I am not sure what that slider is in the tile but it does NOT work but the slider works in HA entity..


```
                     position: [5.32, -1.02],
                     width: 2,
                     height: 1,
                     title: '      Living Room TV',
                     id: 'media_player.living_room_tv_kd_75x8500e',
                     type: TYPES.MEDIA_PLAYER,
                     hideSource: true,
                     hideMuteButton: false,
                     state: false,
                    //state: '@attributes.media_title',
                     subtitle: '@attributes.media_title',
                     bgSuffix: '@attributes.entity_picture',
                     hidden: function (item, entity) { return entity.state == 'off' }
```

**Comment by akloeckner** [comment]
:-\ That should be the volume slider...

**Comment by oneseventhree** [comment]
> :-\ That should be the volume slider...

yeah it should be but I can't drag it on my computer OR on my tablet(which is where I need it to work) so I am unsure what the go is

**Comment by akloeckner** [comment]
Ok. That seems to be too deep in for me. First things that come to my mind:
* Tell us what device and browser you're using.
* Try to comment out the `stopPropagation` lines (one by one) in `index.html` and tell us if things change (hopefully improve):
```
               <tr ng-if="shouldShowVolumeSlider(entity) && (_c = getVolumeConf(item, entity))">
                  <td colspan="3" class="media-player-table--td-volume">
                     <div class="media-player--volume">
                        <input type="range" ng-model="_c.value"
                               ng-change="volumeChanged(item, entity, _c)"
                               ng-on-touchstart="$event.stopPropagation()"
                               ng-on-touchmove="$event.stopPropagation()"
                               ng-on-pointerdown="$event.stopPropagation()"
                               step="{{ _c.step }}" min="{{ _c.min }}" max="{{ _c.max }}">
                     </div>
                  </td>
               </tr>
```

**Comment by oneseventhree** [comment]
it works on mylaptop browers. Sadly not on my tablet. IS there a setting I am missing to enable on the tablet?

**Comment by rchl** [comment]
What browser name and version are you using on your tablet?
It likely has to do with different degree of support for touch/pointer events.

**Comment by oneseventhree** [comment]
> What browser name and version are you using on your tablet?
> It likely has to do with different degree of support for touch/pointer events.

I am using FullyKiosk on a Samsung tablet

**Comment by rchl** [comment]
I've just tried the latest version of Fully kiosk browser with a MEDIA_PLAYER tile and saw no issues.

The issue could be either with your Fully settings (maybe try resetting if there is an option for that) or with old version of WebView that Fully is using (apparently on some tablets it's not able to use latest version).
